sdk/communication/Azure.Communication.CallAutomation/src/CallMedia.cs

```diff
@@ -463,18 +471,5 @@ public virtual Response StopContinuousDtmfRecognition(CommunicationIdentifier ta
                 throw;
             }
         }
-
-        private static SendDtmfRequestInternal CreateSendDtmfRequest(SendDtmfOptions sendDtmfOptions)
```

Comment:
Question: is there a reason why option class have been removed?

Reply:
As per [this](https://github.com/Azure/azure-sdk-for-net/pull/35242/files/192632cec82e3025d2f14dd0fb3fbac9446f7493#r1153796703) review comment from last PR, we removed options from startContinuousDtmfRecognition(), so for consistency it was thought to remove options from sendDtmf as well and this [workitem](https://skype.visualstudio.com/SPOOL/_workitems/edit/3224670) was created to remove options. Additionlly, we don't expect the arguments to grow hence we kept the method simple at this point without options.

If you suggest for options, we will add in next pr as this change will require a contract and a PMA change also.

Reply:
Ah ok - just curiosity, is OperationContext required field here? for all other operations, this is optional value

Reply:
It is not...instead of overloading another method, I have mentioned in method doc to pass null, if not applicable

Reply:
If it is optional value, we would have signature like this 
..., OperationContext = null, ...

This sets default value for OperationContext, and if the devs never pass this value in the first place, SDK will set it as default value. This has few benefits:

1. It clearly shows in signature that which values are required and which values are not required
2. Does not force the devs to pass null when devs do not care for this value
3. Without reading documentation, it gives devs idea that this value's default value is null
4. This aligns with all other methods in our SDK

By making OperationContext optional, would it break your change? if not, I suggest we make this value optional.

Reply:
Sure, done...Thank you